docs/develop/showcase.md

```diff
@@ -0,0 +1,96 @@
+---
+id: showcase
+title: Showcase
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+
+
+## Wallets
+
+Wallets are like the swiss army knife of the web3.
+Your wallet will store private information like your secret for your DID or your KILT credentials.
+Each time a website wants to know something about you, the wallet will ask for your permission to share or sign information.
+
+
+<a href="https://sporran.org">
+    <ThemedImage
+    width="200"
+    alt="Sporran Logo"
+    sources={{
+        light: '/img/showcase/sporran_light.svg',
+        dark: '/img/showcase/sporran_dark.svg',
+    }}
+    />
+</a>
+
+Sporran is the first wallet with KILT support.
+It is maintained by _BOTLabs Trusted Entity B.T.E GmbH_ and the [source code is open source](https://github.com/BTE-Trusted-Entity/sporran-extension).
+
+## Apps
+
+<a href="https://w3n.id">
+    <ThemedImage
+    width="200"
+    alt="Web3Name Logo"
+    sources={{
+        light: '/img/showcase/w3n_light.svg',
```

Comment:
The logos are aligned differently to the left. Just a nice to have, but could you change the svgs, so that the viewbox fits the image? That should resolve it.

Reply:
<img width="318" alt="Screenshot 2022-05-20 at 14 43 08" src="https://user-images.githubusercontent.com/1248214/169530590-3739fbc0-16de-4308-ace7-d8f9da44f204.png">

Reply:
Fixed!